.npmignore

```diff
@@ -1,6 +1,12 @@
-node_modules/
-src/
-build/
+node_modules
+src
+build
 .arcconfig
-.babelrc
 *.yml
+.editorconfig
+karma.config.js
+travis.yml
+CHANGELOG.md
+CONTRIBUTING.md
+lib/test
```

Comment:
The `.npmignore` is for when you publish the package to `npm`. These files shouldn't be needed when someone `npm install` it in a project.